src/Entity/AbstractValuation.php

```diff
@@ -0,0 +1,248 @@
+<?php
+
+namespace Jedkirby\ValPal\Entity;
+
+abstract class AbstractValuation
+{
+    /**
+     * @var string
+     */
+    protected $minValuation;
+
+    /**
+     * @var string
+     */
+    protected $valuation;
+
+    /**
+     * @var string
+     */
+    protected $maxValuation;
+
+    /**
+     * @var string
+     */
+    protected $propertyType;
+
+    /**
+     * @var string
+     */
+    protected $tenure;
+
+    /**
+     * @var string
+     */
+    protected $bedrooms;
+
+    /**
+     * @var string
+     */
+    protected $propertyConstructionYear;
+
+    /**
+     * @var string
+     */
+    protected $minRentalValuation;
+
+    /**
+     * @var string
+     */
+    protected $rentalValuation;
+
+    /**
+     * @var string
+     */
+    protected $maxRentalValuation;
+
+    /**
+     * @param string $value
+     */
+    public function setMinValuation($value)
+    {
+        $this->minValuation = $value;
+    }
+
+    /**
+     * @param string $value
+     */
+    public function setValuation($value)
+    {
+        $this->valuation = $value;
+    }
+
+    /**
+     * @param string $value
+     */
+    public function setMaxValuation($value)
+    {
+        $this->maxValuation = $value;
+    }
+
+    /**
+     * @param string $type
+     */
+    public function setPropertyType($type)
+    {
+        $this->propertyType = $type;
```

Comment:
`propertyType` actually differs to `type`. You're not able to modify the `type` of a valuation, at least, not via a setter.